packages/rich-text/src/plugins/EmbeddedEntityBlock/Util.ts

```diff
@@ -20,7 +26,7 @@ export async function selectEntityAndInsert(nodeType, sdk, editor, logAction) {
     }
     Transforms.select(editor, selection);
     insertBlock(editor, nodeType, entity);
-    logAction('insert', { nodeType });
+    logAction?.('insert', { nodeType });
```

Comment:
Based on the function type the `logAction` can't be undefined?

Reply:
I think I forgot to remove `?.` in this place.